src/libraries/Common/src/System/Collections/Generic/BitHelper.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace System.Collections.Generic
 {
-    internal ref struct BitHelper
+    internal readonly ref struct BitHelper
```

Comment:
Is there a reason _not_ to have readonly here?

Reply:
It doesn't help or hurt anything, given its implementation. With a different implementation, where some or all of the bit state is stored in the instance, it wouldn't be appropriate.  Logically given its surface area it doesn't make sense for it to be readonly, but it's also internal, so, doesn't really matter either way.

Reply:
How would you prefer I leave it?

Reply:
I'd prefer not readonly, as it's confusing seeing a readonly struct with mutating methods (e.g. MarkBit).